src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -3,7 +3,8 @@ import {
   AbstractAppConfig,
   CaseEditorConfig
 } from '@hmcts/ccd-case-ui-toolkit';
-import { environment as mainConfig } from '../environments/environment';
```

Comment:
If you are no longer importing ../enviroments/environment , where are the idam login page and cookie names being configured?

Reply:
this has been moved now... we are still importing environment for you idam login and cookie but have removed the ccd stuff for it thats why you see this... 
in other words it has only been removed tom case.config.ts but not the app

Reply:
Good point !